Tests/Realm.Tests/Database/CollectionTests.cs

```diff
@@ -611,7 +611,7 @@ public void Results_GetFiltered_WhenPredicateIsInvalid_Throws()
         {
             Assert.That(
                 () => _realm.All<A>().Filter("Foo == 5"),
-                Throws.TypeOf<RealmException>().And.Message.Contains("No property 'Foo' on object of type 'A'"));
+                Throws.TypeOf<RealmException>().And.Message.Contains("'class_A' has no property: 'Foo'"));
```

Comment:
This is a regression - the change in wording is fine, but we shouldn't be surfacing the internal naming of tables. Might need to ping @ironage or @jedelbo about the best way to convert these error messages to use the expected type names.

Reply:
we missed a few places, it should be fixed by https://github.com/realm/realm-core/pull/4290

Reply:
Thank you. Then I'll update core in our SDK to point to the right commit when the PR 4290 is merged. And of course change the string in the unit test.